alpha-build-machinery/make/targets/openshift/crd-schema-gen.mk

```diff
@@ -4,23 +4,27 @@ CRD_SCHEMA_GEN_APIS ?=$(error CRD_SCHEMA_GEN_APIS is required)
 CRD_SCHEMA_GEN_MANIFESTS ?=./manifests
 CRD_SCHEMA_GEN_OUTPUT ?=./manifests
 
-crd_patches :=$(subst $(CRD_SCHEMA_GEN_MANIFESTS),$(CRD_SCHEMA_GEN_OUTPUT),$(wildcard $(CRD_SCHEMA_GEN_MANIFESTS)/*.crd.yaml-merge-patch))
-
 # $1 - crd file
 # $2 - patch file
 define patch-crd
 	$(YQ) m -i '$(1)' '$(2)'
 
 endef
 
+# $1 - CRD_SCHEMA_GEN_MANIFESTS
+# $2 - CRD_SCHEMA_GEN_OUTPUT
+define crd-patches
```

Comment:
why moving this into macro? seems like it was doing the same before and it is instantiated in only one place

Reply:
I see why now, this would be the fix https://github.com/openshift/library-go/pull/527/files#r326495025